apps/editing-toolkit/editing-toolkit-plugin/wpcom-block-editor-nux/src/welcome-tour/tour-steps.js

```diff
@@ -75,10 +149,7 @@ function getTourSteps( localeSlug, referencePositioning ) {
 			referenceElements: referencePositioning && referenceElements[ 0 ],
 			meta: {
 				heading: __( 'Welcome to WordPress!', 'full-site-editing' ),
-				description: __(
-					'Take this short, interactive tour to learn the fundamentals of the WordPress editor.',
-					'full-site-editing'
-				),
+				descriptions: getTourDescriptions( 'welcome' ),
 				imgSrc: getTourAssets( 'welcome' ),
```

Comment:
Not a fan of having to repeat the id. Also with [59462](https://github.com/Automattic/wp-calypso/pull/59462) I'll be introducing the id field.
Maybe we can call something like
```
meta: {
				heading: __( 'Welcome to WordPress!', 'full-site-editing' ),
                ...getTourAssetsAndDescriptions( 'welcome' ),
```

? `getTourAssetsAndDescriptions` to be defined 😄

Reply:
> Not a fan of having to repeat the "id".
Me neither, and I would say that we remove the function and add the `descriptions` object to the `meta` object directly., like we did with `referenceElements` (see [#59498](https://github.com/Automattic/wp-calypso/pull/59498/files#diff-31afa086e8ebaae38d27ff5a3afffbcb10eccad6fec5ac0c8cd62b7ea50573bfR72-R77)).

The signature of would look something like this:

```js
{
    description: {
        desktop: string; 
        mobile?:string;
    }
}
```

WDYT?

Reply:
Thanks for that, pushed a new commit: [fe5dcb9](https://github.com/Automattic/wp-calypso/pull/59488/commits/fe5dcb98f94a80012720fba5ce3d3e16566c733c) 🚀

Reply:
Absolutely! That was a typo from my side 😅